sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionErrorsSuite.scala

```diff
@@ -751,8 +751,8 @@ class QueryExecutionErrorsSuite
 
         checkError(
           exception = e.getCause.asInstanceOf[SparkSQLFeatureNotSupportedException],
-          errorClass = "UNSUPPORTED_FEATURE.JDBC_TRANSACTION",
-          parameters = Map.empty)
+          errorClass = "UNSUPPORTED_FEATURE.MULTI_ACTION_ALTER",
+          parameters = Map("tableName" -> "\"test\".\"people\""))
```

Comment:
Any table name should be quoted by backticks. Why it is quoted by ""?.

Reply:
I think because the table name in this test is specified as `"test"."people"` in SQL for JDBC as below:

```scala
      try {
        conn = DriverManager.getConnection(url, new Properties())
        conn.prepareStatement("""CREATE SCHEMA "test"""").executeUpdate()
        conn.prepareStatement(
          """CREATE TABLE "test"."people" (name TEXT(32) NOT NULL, id INTEGER NOT NULL)""")
          .executeUpdate()
        conn.commit()
      } finally {
        if (null != conn) {
          conn.close()
        }
```

I tried to use backticks and other quotations, but it was not working.
Seems like it's only allowed `" "` for JDBC prepareStatement?

Reply:
Seems like the quoting depends on dialects, for example `quoteIdentifier()` quotes by backticks in the case of MySQL.

Reply:
Yeah, I think so.

Do we want to also have similar function to replace double quotes to backticks for JDBC table and column names ?? or leave is as is for now??

Reply:
> or leave is as is for now??

Let's leave this as is.